### ISSUE TITLE
A questions about  seq2seq-torch.py in the 43th line

### ISSUE THREAD

**Issue by MowanHu** [graykode/nlp-tutorial#24]
Hi, Im an nlp rookie, I want to ask u a question, your code extract input(context) in a fixed window in 43th area,  and "word sequence" is a sentences list , some words may extract their neighbour words form different sentences, so, is this way harm to the result?

And my training result seems not very well and I didn't change the codes.
![image](https://user-images.githubusercontent.com/43491475/56549940-c714e780-65b6-11e9-9df5-f07eb412fbbf.png)

If u see this issues, please answer me in your free time. 
Although my english is poor, I still want to express my gratitude to u.

**Comment by graykode** [comment]
Hello. It will be more helpful for me if you add code line link such as https://github.com/graykode/nlp-tutorial/blob/master/4-2.Seq2Seq(Attention)/Seq2Seq(Attention)-Torch.py#L10

**Comment by MowanHu** [comment]
> Hello. It will be more helpful for me if you add code line link such as https://github.com/graykode/nlp-tutorial/blob/master/4-2.Seq2Seq(Attention)/Seq2Seq(Attention)-Torch.py#L10

Sorry, i made a wrong title. The question is from https://github.com/graykode/nlp-tutorial/blob/master/1-2.Word2Vec/Word2Vec-Skipgram-Torch(Softmax).py#L43
And "word sequence" is a sentences list , some words may extract their neighbour words form different sentences, so, is this way harm to the result?

**Comment by graykode** [comment]
Yes you are right but I dont care in this example.

**Comment by MowanHu** [comment]
> Yes you are right but I dont care in this example.

Thank u, I know u means.
